src/encoded/static/components/item-pages/components/donor-overview/ProtectedDonorViewDataCards.js

```diff
@@ -2,6 +2,208 @@
+                            </td>
+                            <td className="text-left">
+                                <b>Ventilator Case.</b> All cases on a
+                                ventilator immediately before death.
+                            </td>
+                        </tr>
+                        <tr>
+                            <td className="fs-5 align-middle text-center px-5">
+                                1
+                            </td>
+                            <td className="text-left">
+                                <b>Violent and fast death.</b> Deaths due to
+                                accident, blunt force trauma or suicide,
+                                terminal phase estimated at &lt; 10 min.
+                            </td>
+                        </tr>
+                        <tr>
+                            <td className="fs-5 align-middle text-center px-5">
+                                2
+                            </td>
+                            <td className="text-left">
+                                <b>Fast death of natural causes.</b> Sudden
+                                unexpected deaths of people who had been
+                                reasonably healthy, after a terminal phase
+                                estimated at &lt; 1 hr (with sudden death from a
+                                myocardial infarction as a model cause of death
+                                for this category).
+                            </td>
+                        </tr>
+                        <tr>
+                            <td className="fs-5 align-middle text-center px-5">
+                                3
+                            </td>
+                            <td className="text-left">
+                                <b>Intermediate death.</b> Death after a
+                                terminal phase of 1 to 24 hrs (not classifiable
+                                as 2 or 4); patients who were ill but death was
+                                unexpected.
+                            </td>
+                        </tr>
+                        <tr>
+                            <td className="fs-5 align-middle text-center px-5">
+                                4
+                            </td>
+                            <td className="text-left">
+                                <b>Slow death.</b> Death after a long illness,
+                                with a terminal phase longer than 1 day
+                                (commonly cancer or chronic pulmonary disease);
+                                deaths that are not unexpected
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+            </PopoverBody>
+        </Popover>
+    );
+}
+
+/**
+ * Popover for Alcohol Exposure Frequency Description.
+ */
+function renderAlcoholFrequencyDescriptionPopover() {
```

Comment:
Similar to Hardy Scale above, let's define the alcohol frequency as an export variable.